### ISSUE TITLE
Removing vPool fails due to 'destroy_filesystem' call core dumping

### ISSUE THREAD

**Issue by kvanhijf** [openvstorage/volumedriver#309]
```
In [8]: path1
Out[8]: 'arakoon://config/ovs/vpools/24c527fe-1dbe-4b26-832a-011387093539/hosts/integrationtests-vpool8Msef832NceGqLJ6/config?ini=%2Fopt%2FOpenvStorage%2Fconfig%2Farakoon_cacc.ini'

In [26]: sd = LocalStorageRouterClient(path1)

In [27]: sd.destroy_filesystem()
/usr/bin/ovs: line 285:   682 Aborted                 (core dumped) ipython
root@ovs01-211:~#
```

Issue is related to this
```
2017-04-11 14:46:34 985917 +0200 - ovs01-211 - 4263/0x00007f2c88fb6700 - LoggerToolCut/AlbaProxyClient - 000000000000002a - error - logMessage: libgcrypt has not been initialized
```

Should be enough information for you @redlicha ?

**Comment by redlicha** [comment]
The alba client lib decides to invoke `abort()` if -lgcrypt is not initialized (as is the case in the Python DSOs):
```
Program terminated with signal SIGABRT, Aborted.
#0  0x00007f73a3e09428 in __GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:54
54	../sysdeps/unix/sysv/linux/raise.c: No such file or directory.
[Current thread is 1 (Thread 0x7f73a45cd700 (LWP 1806))]
(gdb) bt
#0  0x00007f73a3e09428 in __GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:54
#1  0x00007f73a3e0b02a in __GI_abort () at abort.c:89
#2  0x00007f739ab8d1a5 in alba::proxy_client::RoraProxy_client::RoraProxy_client (this=0x10f1c10, delegate=..., rora_config=...) at ../src/lib/rora_proxy_client.cc:43
#3  0x00007f739ab89e1c in alba::proxy_client::make_proxy_client (ip="10.100.199.214", port="26205", timeout=..., transport=@0x7ffcb31e433c: alba::transport::Kind::tcp, rora_config=...)
    at ../src/lib/proxy_client.cc:57
#4  0x00007f739aaed968 in backend::albaconn::Connection::Connection (this=0x1075d60, host="10.100.199.214", port=<optimized out>, timeout=<optimized out>, preset=..., transport=alba::transport::Kind::tcp, 
    rora_config=...) at /opt/vmachines/jenkins_work/workspace/volumedriver-no-dedup-release-ubuntu-16.04/volumedriver-core/src/backend/Alba_Connection.cpp:172
#5  0x00007f739aaee1a4 in backend::albaconn::Connection::Connection (this=0x1075d60, cfg=...)
    at /opt/vmachines/jenkins_work/workspace/volumedriver-no-dedup-release-ubuntu-16.04/volumedriver-core/src/backend/Alba_Connection.cpp:132
#6  0x00007f739aaadf00 in std::make_unique<backend::albaconn::Connection, backend::AlbaConfig const&> () at /usr/include/c++/5/bits/unique_ptr.h:765
#7  backend::ConnectionPool::make_one_ (this=this@entry=0x12b98e0) at /opt/vmachines/jenkins_work/workspace/volumedriver-no-dedup-release-ubuntu-16.04/volumedriver-core/src/backend/ConnectionPool.cpp:103
#8  0x00007f739aaaec7a in backend::ConnectionPool::get_connection (this=0x12b98e0, force_new=force_new@entry=ForceNewConnection::F)
    at /opt/vmachines/jenkins_work/workspace/volumedriver-no-dedup-release-ubuntu-16.04/volumedriver-core/src/backend/ConnectionPool.cpp:157
#9  0x00007f739aa8088c in backend::BackendConnectionManager::getConnection (this=this@entry=0x12b8c40, force_new=force_new@entry=ForceNewConnection::F, nspace=...)
    at /opt/vmachines/jenkins_work/workspace/volumedriver-no-dedup-release-ubuntu-16.04/volumedriver-core/src/backend/BackendConnectionManager.cpp:129
#10 0x00007f739aa9638f in backend::(anonymous namespace)::NamespaceConnFetcher::operator() (cm=..., nspace=..., attempt=attempt@entry=0, this=<optimized out>)
    at /opt/vmachines/jenkins_work/workspace/volumedriver-no-dedup-release-ubuntu-16.04/volumedriver-core/src/backend/BackendInterface.cpp:132
#11 0x00007f739aa9b174 in backend::BackendInterface::do_wrap_<backend::(anonymous namespace)::NamespaceConnFetcher, void, backend::Namespace const&> (this=0x10754c0, params=..., mem_fun=<optimized out>, 
    get_conn=...) at /opt/vmachines/jenkins_work/workspace/volumedriver-no-dedup-release-ubuntu-16.04/volumedriver-core/src/backend/BackendInterface.cpp:67
[...]
```

**Comment by JeffreyDevloo** [comment]
Fixed in the latest OSE 6.14

**Comment by redlicha** [comment]
6.15.0.*dev* builds - official release will be 6.16.0
